### PR TITLE
azure-rules: check that /etc/sudoers set up correctly

### PR DESCRIPTION
**Pull request by aleksei-burlakov**
Ref: 8.3 in https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/

### CODE REVIEW

File: examples/azure-rules.yaml [hunk 1]
```diff
@@ -656,4 +656,67 @@ groups:
...
+      - id: 1.6.5
+        description: "Check SFAIL_SITEB alias"
+        audit: 'cat /etc/sudoers | grep -E "^Cmnd_Alias +SFAIL_SITEB += +/usr/sbin/crm_attribute" | grep -E "\-n +hana_.+_site_srHook_.+" | grep -E "\-v +SFAIL" | grep -E "\-t +crm_config" | grep -E "\-s +SAPHanaSR" | sed "s/.*/true/"'
+        tests:
+          test_items:
+          - flag: "true"
+        remediation: |
+          Set up the Cmnd_Alias in /etc/sudoers.
+          Ref: section 8.3 in https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/ 
+        scored: true
```
Comment (stefanotorresi) [inline examples/azure-rules.yaml:712]: The remediation for these looks a bit vague and ambiguous, could you make it more specific? ;)

**Review by stefanotorresi — CHANGES_REQUESTED** [review]

File: examples/azure-rules.yaml (hunk 1 above)
Comment (aleksei-burlakov) [reply examples/azure-rules.yaml]: Done!

File: examples/azure-rules.yaml [hunk 2]
```diff
@@ -656,4 +656,67 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check that sap sudo user is set up in /etc/sudoers"
```
Comment (dirkmueller) [inline examples/azure-rules.yaml:661]:
```suggestion
    description: "Check that <sidadm> sudo user is set up in /etc/sudoers"
```

Comment (aleksei-burlakov) [reply examples/azure-rules.yaml]: Thanks, changed.

**Review by dirkmueller — APPROVED** [review]

**Review by stefanotorresi — APPROVED** [review]
LGTM

**Pull request merged**